### ISSUE TITLE
The loss on COCO dataset of training stage1 dosen't decrease

### ISSUE THREAD

**Issue by habbakuk1103** [layumi/Image-Text-Embedding#5]
Hi, I run the train_coco_word2_1_pool.m, but after more than 10 epochs, I found  the train result is still bad(as below). I didn't change any hyper parameters but I don't know why it doesn't work.
![qq 20180515115715](https://user-images.githubusercontent.com/28893002/40035968-3af3de2e-5837-11e8-9f66-24f1cd071c1a.png)
Can you tell me what make this result happen? And I found the learning rate in your code is 0.1, but it's reported as 0.001 in the paper. Which lr is correct and better in this task?

**Comment by layumi** [comment]
The learning rate = global learning rate * layer learning rate. 
For COCO, it may converge slowly. It may take about 20 epochs to converge.

**Comment by layumi** [comment]
I also notice there is error in your code? @habbakuk1103 
Do you prepare the dataset in the right way?

**Comment by habbakuk1103** [comment]
Thank you for response.
 I think I prepare the COCO dataset as you told, but the dimension of my 'coco_dictionary.mat' is 29141, but it's 29972 in your code. I don't know whether it's caused by I prepare the dataset in the matlab r2016b on Windows system of my computer because I can't install this  version on our linux server. Could you  give me a link to download the data after preprocessed as below if possible?
![qq 20180515202323](https://user-images.githubusercontent.com/28893002/40056428-f7f4691c-587d-11e8-8ed5-8a0578155ae8.png)
By the way, I am not familiar with matconvnet. Could you tell me what the 'error' in the result mean? I check the code before and I think the data and label should be aligned.

**Comment by layumi** [comment]
![image](https://user-images.githubusercontent.com/8390471/40091654-89c9a538-58eb-11e8-8df5-20cc56d12a89.png)

I wonder whether you provided the right file path.

**Comment by habbakuk1103** [comment]
The path of the  'prepare_imdb.m' is as below in matlab on my server, but its version is matlab R2014b which doesn't have 'jsondecode' function
![1](https://user-images.githubusercontent.com/28893002/40100682-6b83bb50-5917-11e8-8f2c-4755b6353267.png)
So I implement the program on my computer, thus the path is changed as below
![2](https://user-images.githubusercontent.com/28893002/40100787-d1b40858-5917-11e8-9d85-63bd6ea37609.png)
I wonder whether this difference of paths cause the problem.

**Comment by habbakuk1103** [comment]
I have solved the problem. Thank you for your suggestion.
